user-guide/modules/ROOT/pages/task-simulation.adoc

```diff
@@ -18,4 +18,107 @@ Creating a real-time simulation of vehicles involves various aspects, including
 
 * boost:mpi[] or boost:asio[]: For distributed simulations that run across multiple systems, you might need a library for network communication. boost:mpi[] provides a pass:[C++] interface for the Message Passing Interface (MPI) standard for distributed computing. boost:asio[] can also handle networking tasks and it is a bit lower-level.
 
-* boost:serialization[]: To save the state of the simulation or to communicate complex data structures over a network, you might find this library helpful.
\ No newline at end of file
+* boost:serialization[]: To save the state of the simulation or to communicate complex data structures over a network, you might find this library helpful.
+
+== Build only the Specific Libraries
```

Comment:
This looks like a huge section repeating the content of "Getting Started". Assuming it's a good idea for "scenarios" pages to repeat this content, why would it be only about `task-simulation.adoc` and not other "scenarios"?